dynamodump/dynamodump.py

```diff
@@ -582,6 +583,13 @@ def do_empty(dynamo, table_name):
     table_global_secondary_indexes = table_desc.get("GlobalSecondaryIndexes")
 
     optional_args = {}
+    if billing_mode == "PROVISIONED":
```

Comment:
Use the variable defined rather than just arbitrary strings?

Reply:
I'm not sure about that but i commited a new proposition, let me know what do you think about